content/en/docs/concepts/signals/logs.md

```diff
@@ -5,31 +5,167 @@ description: A recording of an event.
+the OpenTelemetry specification. For the individual language specific
+implementations of the Logs API & SDK, the status is as follows:
+
+{{% signal-support-table "logs" %}}
+
+## Structured, Unstructured, and Semistructured Logs
+
+OpenTelemetry does not distinguish between structured and unstructured logs.
+However, it is vitally important to understand the different kinds of formats
+that log data can manifest in.
+
+### Structured logs
+
+A structured log is a log whose textual format follows a consistent,
+machine-readable format. Two of are JSON:
+
+```json
+{
+  "timestamp": "2024-08-04T12:34:56.789Z",
+  "level": "INFO",
+  "service": "user-authentication",
+  "environment": "production",
+  "message": "User login successful",
+  "context": {
+    "userId": "12345",
+    "username": "johndoe",
+    "ipAddress": "192.168.1.1",
+    "userAgent": "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/104.0.0.0 Safari/537.36"
+  },
+  "transactionId": "abcd-efgh-ijkl-mnop",
+  "duration": 200,
+  "request": {
+    "method": "POST",
+    "url": "/api/v1/login",
+    "headers": {
+      "Content-Type": "application/json",
+      "Accept": "application/json"
+    },
+    "body": {
+      "username": "johndoe",
+      "password": "******"
+    }
+  },
+  "response": {
+    "statusCode": 200,
+    "body": {
+      "success": true,
+      "token": "jwt-token-here"
+    }
+  }
+}
+```
+
+and for infrastructure components, Common Log Format (CLF):
+
+```text
+127.0.0.1 - johndoe [04/Aug/2024:12:34:56 -0400] "POST /api/v1/login HTTP/1.1" 200 1234
+```
+
+It is also common to have different structured log formats mixed together. For
+example, and Extended Log Format (ELF) log mixes JSON with the
+whitespace-separated data in a CLF log.
+
+```text
+192.168.1.1 - johndoe [04/Aug/2024:12:34:56 -0400] "POST /api/v1/login HTTP/1.1" 200 1234 "http://example.com" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/104.0.0.0 Safari/537.36" {"transactionId": "abcd-efgh-ijkl-mnop", "responseTime": 150, "requestBody": {"username": "johndoe"}, "responseHeaders": {"Content-Type": "application/json"}}
+```
+
+To make the most use of this log, it is recommended to parse bothe the JSON and
+the ELF-related pieces into a shared format to make analysis on an Observability
+backend easier.
+
+Structured logs are the preferred way to use logs. Because they are emitted in a
+consistent format, they are easier to parse, which makes them easier to
+preprocess in an OpenTelemetry Collector, correlate with other data, and
+ultimate analyze in an Observability backend. Use structured logs if you can.
+
+### Unstructured Logs
+
+Unstructured logs are logs that don't follow a consistent structure. They may be
+more human-readable, and are often used in development. However, it is not
+preferred to use unstructured logs to contain useful debugging data if you can
+help it.
```

Comment:
```suggestion
Unstructured logs are logs that don't follow a consistent structure. They may be
more human-readable, and are often used in development.
```

I think the recommendation is really a preference here, and not something always actionable. Plus, passive voice is a red flag. :)